### PR TITLE
Sams pictures

### PR DESCRIPTION
**Pull request by samalexhall**
Created PictureObject class with constructors (getters and setters)
Author: Sams (plural) sh1157 and sr896

### CODE REVIEW

File: TeachEasy/src/teacheasy/data/PictureData.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	}
+
+	public void setRotation(float nRotation) {
+		this.rotation = nRotation;
+	}
+
+	public float getxScaleFactor() {
+		return xScaleFactor;
+	}
+
```
Comment (ajewers) [inline TeachEasy/src/teacheasy/data/PictureData.java]: Style: Unnecessary White Space

File: TeachEasy/src/teacheasy/data/PictureData.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package teacheasy.data;
```
Comment (ajewers) [inline TeachEasy/src/teacheasy/data/PictureData.java]: Style: Missing Copright Comment before Package line

File: TeachEasy/src/teacheasy/data/PictureData.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package teacheasy.data;
+
+import teacheasy.data.PageObject.PageObjectType;
```
Comment (ajewers) [inline TeachEasy/src/teacheasy/data/PictureData.java:9]: This import should not be necessary? Correct me if wrong

File: TeachEasy/src/teacheasy/data/PictureData.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+/**
+ * A class to hold Picture Data
+ * @author sh1157 sr896
+ *
+ */
+public class PictureData extends PageObject {
+	String url;
+	String uri;
+	float xScaleFactor, yScaleFactor, rotation;
```
Comment (ajewers) [inline TeachEasy/src/teacheasy/data/PictureData.java]: Style: Coding standard recommends one variable declaration per line. It seems reasonable to group variables with related purposes (x and y) but perhaps better to have a new line for rotation?

**Comment by ajewers** [comment]
Code looks good, should behave exactly as needed. Minor style comments above.

**Comment by samalexhall** [comment]
Cool
The random looking commit was changing the class name I think

**Comment by ajewers** [comment]
Ah yes, weird that didn't come up for me before. Great stuff!

**Comment by samalexhall** [comment]
Hooray!!!

**Comment by ajewers** [comment]
Woo haha. Unfortunately i've just noticed a problem i didnt see before. It's the same as the comment i just made in your other branch about how all variables must be initialised to a value in the constructor to avoid null pointer errors.

**Comment by samalexhall** [comment]
:'(

**Comment by ajewers** [comment]
Most of #10

**Pull request merged**